expression/builtin_cast_bench_test.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expression
+
+import (
+	"testing"
+
+	"github.com/pingcap/parser/mysql"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/chunk"
+	"github.com/pingcap/tidb/util/mock"
+)
+
+func BenchmarkCastIntAsInt(b *testing.B) {
```

Comment:
could you post the benchmark code on the old cast implementation?

Reply:
Try to use https://godoc.org/golang.org/x/perf/cmd/benchstat and post the benchmark result in the pull request description

Reply:
It is implemented here (https://github.com/pingcap/tidb/pull/11826/files#diff-2967d2754f43b7d6318020f0953a69f3R38)